airflow/providers/salesforce/hooks/salesforce.py

```diff
@@ -336,3 +348,44 @@ def object_to_df(
             df["time_fetched_from_salesforce"] = fetched_time
 
         return df
+
+    def write_schema_to_file(self, schema_file_name: str, obj: str, record_time_added: bool):
+        """
+        Export BigQuery schema to file
+
+        Write a JSON schema file with data types inferred from the Salesforce client.
+
+        :param schema_file_name: file name of the schema file (e.g schema.JSON)
+        :type schema_file_name: str
+        :param obj: Salesforce object name (e.g. Contact, Asset)
+        :type obj: str
+        :param record_time_added: Wheter to add a time_fetched_from_salesforce column
```

Comment:
`record_time_added` somewhat confusing name

record can imply db record in this context

fetched better than added

maybe just `add_timestamp`?